src/NuGet.Indexing/Extraction/DocumentCreator.cs

```diff
@@ -128,110 +119,101 @@ private static void AddVersion(Document document, IDictionary<string, string> pa
                     }
                     else
                     {
-                        errors.Add("Unable to parse 'originalVersion' property.");
+                        errors.Add("Unable to parse 'originalVersion' as NuGetVersion.");
```

Comment:
Are errors logged somewhere? What happens to the errors variable if we crash out, do we still get them in logs?